.git_archival.txt

```diff
@@ -0,0 +1,8 @@
+# The name of this file is hardcoded in the setuptools_scm_git_archive
```

Comment:
_I think_ this could be tested with `git tag -a v2.0.0 v2.0.0 && git archive && python -m pip install ./that/created/archive.tar.gz && python -m pip show ansible-navigator` (on a fresh detached git clone).